go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/golang/mock v1.3.1
 	github.com/google/uuid v1.1.1
 	github.com/gorilla/mux v1.7.3
+	github.com/mr-tron/base58 v1.1.2
```

Comment:
We are using a different base58 package elsewhere.

Reply:
"github.com/btcsuite/btcutil/base58"

Reply:
Good catch, thought I'd gotten them all.

Reply:
Ah, I did get them all, but I didn't tidy the mod file.